webpack/src/ForemanDiscovery/DiscoveredHosts/Components/EmptyState/EmptyState.js

```diff
@@ -12,7 +12,7 @@ const EmptyState = props => {
   };
   return (
     <ForemanEmptyState
-      header="Foreman Discovery"
+      header={__('Foreman Discovery')}
```

Comment:
Hi,
thanks for the PR, just have one question - why to make `Foreman Discovery` available for the translation?
Do we actually want to translate the plugin name?

Reply:
Well, if you want to replace Foreman with "downstream product" using Locale, this would be necessary.